12 - golang-test-web-app fails to find martini package

```
golang-test-web-app
main.go:7:2: cannot find package "github.com/go-martini/martini" in any of:
	/usr/src/go/src/github.com/go-martini/martini (from $GOROOT)
	/go/src/github.com/go-martini/martini (from $GOPATH)
```

--- a/12_run_tests_before_deploy/pipeline.yml
+++ b/12_run_tests_before_deploy/pipeline.yml
@@ -13,12 +13,8 @@ jobs:
         image: docker:///golang#1.4.2
         inputs:
           - name: resource-web-app
-            path: .
         run:
-          path: go
-          args:
-            - test
-            - ./...
+          path: ./resource-web-app/test
   - aggregate:
     - put: resource-deploy-web-app
       params:

--- a/12_run_tests_before_deploy/pipeline.yml
+++ b/12_run_tests_before_deploy/pipeline.yml
@@ -35,4 +35,3 @@ resources:
     password: {{cf-password}}
     organization: {{cf-organization}}
     space: {{cf-space}}
-    skip_cert_check: {{cf-skip-cert-check}}

--- a/12_run_tests_before_deploy/stub.example.yml
+++ b/12_run_tests_before_deploy/stub.example.yml
@@ -3,4 +3,3 @@ cf-username: EMAIL
 cf-password: PASSWORD
 cf-organization: ORG
 cf-space: SPACE
-cf-skip-cert-check: false

